conda_forge_tick/migrators.xsh

```diff
@@ -600,6 +601,37 @@ class Rebuild(Migrator):
         return n
 
 
+class CompilerRebuild(Rebuild):
+    bump_number = 1000
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        with indir(recipe_dir):
+            with indir('..'):
+                from ruamel.yaml import safe_load, safe_dump
```

Comment:
Is there a reason not to have these imports at module level scope? We are already importing conda after all...